docker-for-mac/release-notes.md

```diff
@@ -16,11 +16,42 @@ This page contains information about the new features, improvements, known issue
 > Starting with Docker Desktop 3.0.0, Stable and Edge releases are combined into a single release stream for all users. Updates to Docker Desktop will now be available automatically as delta updates from the previous version. This means, when there is a newer version of Docker Desktop, it will be automatically downloaded to your machine. All you need to do is to click **Update and restart** from the Docker menu to install the latest update.
 {: .important }
 
+## Docker Desktop Community 3.2.0
+2021-03-NN
+
+> [Download](https://desktop.docker.com/mac/stable/amd64/Docker.dmg)
+
+### New
+
+- Add experimental support to use new virtualization framework (requires macOS 11.1 or later)
+- Use iTerm2 to do `docker exec` if iTerm2 is installed, if not then use the default Terminal.App [docker/for-mac#4273]((https://github.com/docker/for-mac/issues/4273)
+
+### Upgrades
```

Comment:
I’m pretty sure there is Go 1.16 as well, right?

Reply:
Yes, I wasn't sure if it's interesting for the users. OK, we did it in the past, so I will add it.